src/utils/priceAndQuantityHandler.js

```diff
@@ -0,0 +1,18 @@
+export const priceAndQuantityHandler = () => {
+  const getTotalPrice = (cartItems) => {
+    return cartItems.reduce((acc, curr) => {
+      return (acc = acc + Number(curr.originalPrice));
+    }, 0);
+  };
+  const getDiscountedPrice = (cartItems) => {
+    return cartItems.reduce((acc, curr) => {
+      return (acc = acc + Number(curr.discountedPrice));
+    }, 0);
+  };
+  const getTotalQuantity = (cartItems) => {
+    return cartItems.reduce((acc, curr) => {
+      return (acc = acc + Number(curr.qty));
+    }, 0);
+  };
```

Comment:
Suggestion to use single reduce to get all the values

Reply:
Ya I can do that.